resources/prosody-plugins/mod_speakerstats_component.lua

```diff
@@ -0,0 +1,189 @@
+local get_room_from_jid = module:require "util".get_room_from_jid;
+local jid_resource = require "util.jid".resource;
+local ext_events = module:require "ext_events"
+local st = require "util.stanza";
+local socket = require "socket";
+local json = require "util.json";
+
+local muc_component_host = module:get_option_string("muc_component");
+if muc_component_host == nil then
+    log("error", "No muc_component specified. No muc to operate on!");
+    return;
+end
+local muc_module = module:context("conference.damencho.jitsi.net");
```

Comment:
Maybe "conference.damencho.jitsi.net" should be `muc_component_host` or something else?

Reply:
ups